python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -79,6 +87,10 @@ def start_ganga(gangadir_for_test='$HOME/gangadir_testing', extra_opts=[]):
         else:
             logger.info("InternalServices still running")
 
+        # The queues are shut down by the atexit handlers so we need to start them here
+        from Ganga.Core.GangaThread.WorkerThreads import startUpQueues
+        startUpQueues()
```

Comment:
`startUpQueues` is in `initEnvironment` in bootstrap.py so currently it is only run if `do_config` is `True`. If we could move it elsewhere then this would not be needed.

`startUpQueues` has a `if _global_queues is None:` before doing anything so it's safe to run if there are already queues running.